### PR TITLE
[ADD] Use Case : noupdate_xml_change

### PR DESCRIPTION
**Pull request by legalsylvain**

### CODE REVIEW

File: docsource/use_cases/noupdate_xml_entry_changed.rst [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+.. csv-table::
+   :header: "id", "name", "header_spacing", "margin_bottom"
+
+   "28", "US Batch Deposit", "0", "10"
+
+V15 table ``report_paperformat`` (Without migration script)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. csv-table::
+   :header: "id", "header_spacing", "margin_bottom"
```
Comment (remytms) [inline docsource/use_cases/noupdate_xml_entry_changed.rst]: I think csv-table `header` is missing the column `name`.

File: docsource/use_cases/noupdate_xml_entry_changed.rst [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+**Problem**:
+
+- The paperformat doesn't contains the new default values.
+
+V12 table ``ir_act_window_view`` (With migration script)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. csv-table::
+   :header: "id", "header_spacing", "margin_bottom"
```
Comment (remytms) [inline docsource/use_cases/noupdate_xml_entry_changed.rst]: Idem about the missing `name` column.

File: docsource/use_cases/noupdate_xml_entry_changed.rst (hunk 1 above)
Comment (legalsylvain) [reply docsource/use_cases/noupdate_xml_entry_changed.rst]: thanks ! fixed.

File: docsource/use_cases/noupdate_xml_entry_changed.rst (hunk 2 above)
Comment (legalsylvain) [reply docsource/use_cases/noupdate_xml_entry_changed.rst]: also fixed.

**Review by remytms — APPROVED** [review]
LGTM. Very helpfull. :)

File: docsource/use_cases/noupdate_xml_entry_changed.rst [hunk 3]
```diff
@@ -0,0 +1,244 @@
+Noupdate XML entry Changed
+++++++++++++++++++++++++++
+
+Rational
+++++++++
```
Comment (yostashiro) [inline docsource/use_cases/noupdate_xml_entry_changed.rst]:
```suggestion
Rationale
+++++++++
```

**Review by yostashiro — COMMENTED** [review]
Nitpicking.

**Comment by legalsylvain** [comment]
hi @yostashiro, could you update your review ? 
thanks !

**Review by yostashiro — APPROVED** [review]
Thanks for the documentation!

File: docsource/use_cases/noupdate_xml_entry_changed.rst [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+.. csv-table::
+   :header: "id", "name", header_spacing", "margin_bottom"
+
+   "28", "US Batch Deposit", "0", "10"
+
+**Problem**:
+
+- The paperformat doesn't contains the new default values.
+
+V12 table ``ir_act_window_view`` (With migration script)
```
Comment (remi-filament) [inline docsource/use_cases/noupdate_xml_entry_changed.rst]: Should be v15 here :
```suggestion
V15 table ``ir_act_window_view`` (With migration script)
```

**Review by remi-filament — CHANGES_REQUESTED** [review]
Minor typo, thank you @legalsylvain for this clear documentation !

**Review by remi-filament — APPROVED** [review]
Thanks !

**Pull request merged**